client/blocks/inline-help/admin-sections.js

```diff
@@ -0,0 +1,430 @@
+		),
+		link: '/me/account',
+		synonyms: [ 'dashboard', 'change', 'language' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Close my account permanently' ),
+		description: translate( 'Delete all of your sites, and close your account completely.' ),
+		link: '/me/account/close',
+		synonyms: [ 'delete' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Change my account privacy settings' ),
+		link: '/me/privacy',
+		synonyms: [ 'security', 'tracking' ],
+		icon: 'visible',
+	},
+	{
+		title: translate( 'View my purchase and billing history' ),
+		link: '/me/purchases',
+		synonyms: [ 'purchases', 'invoices', 'pending', 'payment', 'credit card' ],
+		icon: 'credit-card',
+	},
+	{
+		title: translate( 'Download the WordPress.com app for my device' ),
+		description: translate( 'Get WordPress apps for all your screens.' ),
+		link: '/me/get-apps',
+		synonyms: [ 'android', 'iphone', 'mobile', 'desktop', 'phone' ],
+		icon: 'my-sites',
+	},
+	{
+		title: translate( 'View my drafted posts' ),
+		link: `/posts/drafts/${ siteSlug }`,
+		synonyms: [ 'posts', 'draft' ],
+		icon: 'my-sites',
+	},
+	{
+		title: translate( 'Manage my blog posts' ),
+		link: '/posts/${ siteSlug }',
+		synonyms: [ 'lists', 'posts' ],
+		icon: 'my-sites',
+	},
+	{
+		title: translate( 'View my drafted pages' ),
+		link: `/pages/drafts/${ siteSlug }`,
+		synonyms: [ 'pages', 'draft' ],
+		icon: 'my-sites',
+	},
+	{
+		title: translate( 'Manage my pages' ),
+		link: `/pages/${ siteSlug }`,
+		synonyms: [ 'lists', 'pages' ],
+		icon: 'my-sites',
+	},
+	{
+		title: translate( 'I cannot find my site on Google' ),
+		link: `/marketing/traffic/${ siteSlug }`,
+		synonyms: [ 'google', 'traffic' ],
+		icon: 'speaker',
+	},
+	{
+		title: translate( 'Verify my site with Google' ),
+		link: `/marketing/traffic/${ siteSlug }`,
+		synonyms: [ 'google', 'traffic' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Create a new site' ),
+		link: '/jetpack/new?ref=calypso-selector',
+		synonyms: [ 'site' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'View contact form messages' ),
+		link: `https://${ siteSlug }/wp-admin/edit.php?post_type=feedback&calypsoify=1`,
+		synonyms: [ 'contact', 'form' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Portfolio projects (for those who have them active)' ),
+		link: `/types/jetpack-portfolio/${ siteSlug }`,
+		synonyms: [ 'portfolio' ],
+		icon: 'cog',
+	},
+] );
+
+/**
+ * Returns a filtered site admin collection.
+ *
+ * @param   {string} searchTerm The search term
+ * @param   {Array}  collection A collection of site admin objects
+ * @param   {number} limit      The maximum number of results to show
+ * @returns {Array}             A filtered (or empty) array
+ */
+export function filterListBySearchTerm( searchTerm = '', collection = [], limit = 4 ) {
+	const searchTermWords = words( searchTerm ).map( ( word ) => word.toLowerCase() );
+
+	if ( searchTermWords.length < 1 ) {
```

Comment:
```suggestion
	if ( ! searchTermWords.length ) {
```

Simplify. If there are no results or the result evaluates to a falsy value. The use of the `<` feels inconsistent and adds cognitive load because I'm having to think "why did they use `<` there must be a good reason".